pkg/nimble/Makefile

```diff
@@ -15,6 +15,7 @@ ifeq (llvm,$(TOOLCHAIN))
 # tell LLVM/clang not to be so pedantic with this.
   CFLAGS += -Wno-unused-function
   CFLAGS += -Wno-sometimes-uninitialized
+  CFLAGS += -Wno-address-of-packed-member
```

Comment:
This should actually not be deactivated, but fix as this can lead to unexpected crashes. I'd rather deactivate building `nrf52dk` with LLVM for now than deactivating this warning.

Reply:
well `example/nimble_gatt` is only build for the `nrf52dk` with llvm anyway, so the effect of the quick fix is the same. See here https://ci.riot-os.org/RIOT-OS/RIOT/10484/262a04c9cfb390f7ae2aead657a2b62a8a6f7e16/output.html and search for `nimble_gatt`.

Reply:
to that end I would rather disable this warning and get the example build with llvm at least once, blacklisting the nrf52dk as you suggested would lead to not compiling this with llvm at all

Reply:
I know, but getting an application built is not the task of a CI builder. It is testing the *correct* build of an application. As such, even as a quick fix I don't except building with such a critical warning disabled, so that at least in a local environment people might get warned. The right way is to temporarily "drop" support and then resupport once the bug is fixed.

Reply:
But maybe someone else has differing opinions, so I'm not blocking because of this, but I won't ACK neither.

Reply:
mhm, agree to disagree here. 

The *bug* was only "found" because we updated the CI workers from xenial to bionic yesterday, so it was there all the time, just that the newer LLVM is a bit more strict. So its either dropping the board (and with that llvm) completely **or** dropping that warning and have llvm compile that thing at least once. And once fixed, re-arm the warning.

I just want to get Murdock "working" again ASAP, to get other (real) PRs merged ...

Reply:
> the board (and with that llvm) completely 

Not true. You can [disable just `nrf52dk`](https://github.com/RIOT-OS/RIOT/blob/fdde802bd7fb258588bdbc3317a1ac40d0990aea/.murdock#L4). Yes, we drop LLVM support with that for that particular board, but not for all boards (we don't build with LLVM for all boards anyway).

Reply:
The bug is also present in the gcc build. gcc just doesn't warn.
It also doesn't matter on the nrf52dk, as Cortex M4 supports unaligned accesses (they're just a tad slower). I've checked the code, there are only unaligned accesses when some debug mode is activated, the regular case ends up using memcpy.

Thus I think enabling the warning is fine...